docker-compose.background.yml

```
@@ -28,13 +28,11 @@ services:
     volumes:
       - ./data/redis:/data
     command: redis-server --appendonly yes
-  # To enable the HL7 Validator Wrapper, both the section below and
-  # the section in nginx.background.conf need to be uncommented
-  # hl7_validator_service:
-  #   image: infernocommunity/inferno-resource-validator
-  #   # Update this path to match your directory structure
-  #   volumes:
-  #     - ./igs:/home/igs
-  #     # To let the service share your local FHIR package cache,
-  #     # uncomment the below line
-  #     # - ~/.fhir:/home/ktor/.fhir
\ No newline at end of file
+  hl7_validator_service:
+    image: infernocommunity/inferno-resource-validator
+    # Update this path to match your directory structure
+    volumes:
+      - ./igs:/home/igs
```

I think this was originally my copy & paste error coming from the `validator_service` above, but this volume should point to /app/igs
```suggestion
      - ./igs:/app/igs
```
(for example see the carin4bb test kit which needs to read from file https://github.com/inferno-framework/carin-for-blue-button-test-kit/blob/main/docker-compose.background.yml#L10 )